### ISSUE TITLE
[NEW BUILD - DEFAULT / 19w22g] Hotfix never dies

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Skywanderers#1163]
More tweak linked to RTX. Also now I output the version id in the log as I am pushing a lot of hotfix. Thanks

**Comment by Tommyg930** [comment]
well it no longer stays stuck in the loading screen as long, still CTD after the crash handler pops up for a brief moment.

[output_log.txt](https://github.com/tsunamayo/Skywanderers/files/3243449/output_log.txt)

Is it fine that I'm just replying to you here?

**Comment by tsunamayo** [comment]
Here is fine! Could you please send what is inside the folder:
 C:/Users/XXXXX/AppData/Local/Temp/TsunamayoCorp/Skywanderers/Crashes
Thanks!

**Comment by Tommyg930** [comment]
Sure thing, here ya go.
[Crash_2019-06-01_061930237.zip](https://github.com/tsunamayo/Skywanderers/files/3243480/Crash_2019-06-01_061930237.zip)

**Comment by Tommyg930** [comment]
BTW, while the loading screen remains. I do get a response if I press escape. I hear the escape menu open and close and when its open the mouse cursor comes on screen, disappearing again when the escape menu closes. I did manage to blind click the 'exit game' button.

![skywanderers](https://user-images.githubusercontent.com/33021337/58745763-48cd2000-8455-11e9-90a8-d4cb346227ee.png)

**Comment by tsunamayo** [comment]
Could you please try before launching the game to desactivate Bloom and Ambient Occlusion (setting from main menu). I will do another build test, but after I am afraid I will have to check up with the Unity guys, as it seems pretty low level stuff. Thanks

**Comment by Tommyg930** [comment]
Funny enough. I had thought to try that myself. the loading screen is still visible but I am getting audio feed back from pressing Esc and the mouse cursor appears on screen. going away when I press it a second time and I managed to blind click the exit game button.

No CTD or the crash handler popping up this time. but at the same time, no visuals

[output_log.txt](https://github.com/tsunamayo/Skywanderers/files/3243605/output_log.txt)

<img width="767" alt="e9576af95d5f6f01cefba16b5552babc" src="https://user-images.githubusercontent.com/33021337/58746192-abc1b580-845b-11e9-9c39-028924e0c1ab.png">
![758a8ddb13fc48444f85afd1fbcc8545](https://user-images.githubusercontent.com/33021337/58746207-dc095400-845b-11e9-8d0d-53c8b44a4964.jpg)

**Comment by tsunamayo** [comment]
For some reason your GFX card cant create ressources. I will upgrade my unity build version and keep you posted. Thanks!

**Comment by Tommyg930** [comment]
Well that is a bummer.
I did do some searching on my own and I've seen a few instances of people getting the same error messages. Dunno what version of unity you are using but from what I've read unity versions 2018.3.4 and below have the same issue with RTX cards, with some saying that the issue being fixed in Unity 2018.3.5


I'll keep monitoring this thread in case you need me to test something out. that or you can hit me up on the Skywanderers Discord: Ferro#0481

**Comment by tsunamayo** [comment]
yes there is hope, I am on Unity 2018.3.3, and will upgrade to 2018.13.

**Comment by Tommyg930** [comment]
Keeping my fingers crossed then ^_^

**Comment by tsunamayo** [comment]
New updated Unity version is live! Tell if how it goes. Cheers

**Comment by Tommyg930** [comment]
Ok so 19w22h, no luck on that one either.
[output_log.txt](https://github.com/tsunamayo/Skywanderers/files/3243691/output_log.txt)

Is there a command line argument I can use that will force it to use d3d9 or d3d12 instead of d3d11? just throwing out idea's

**Comment by tsunamayo** [comment]
=( No I need dx3d11 only. Do you happen to have a G-sync and/or dual monitor?

**Comment by Tommyg930** [comment]
Don't know what G-sync is, but yeah I am running 3 screens, lemme guess, you want me to shut off the 2 extra screens?

**Comment by tsunamayo** [comment]
Yes that would be good test to do. https://en.wikipedia.org/wiki/Nvidia_G-Sync 
With G-sync you sync your monitor framerate with your nvidia, you need a compatible (more expensive) monitor for that.

**Comment by Tommyg930** [comment]
Yeah I don't have G-sync, I'm using regular samsung flat screen televisions for monitors. 1x 43" 2x 32".

Just did a test where disabled the extra screens. only change is that it now does a CTD after flashing the crashhandler really quick



[output_log.txt](https://github.com/tsunamayo/Skywanderers/files/3243721/output_log.txt)

[Crash_2019-06-01_114911307.zip](https://github.com/tsunamayo/Skywanderers/files/3243724/Crash_2019-06-01_114911307.zip)

**Comment by tsunamayo** [comment]
Okay last few test, reduce resolution and run the game with v-sync (from the game graphic options). Do you have any problem with other games? Thanks

**Comment by Tommyg930** [comment]
Switched it down to 1920 x 1080 @ 30Hz.
Fullscreen, V-sync & anti-aliasing = on
Ambient occlusion, bloom and flare, chromatic aberration & vignetting = off

Now it takes significantly longer to load the game until I hear the esc menu snap. but I'm still only seeing the loading screen and it doesn't CTD nor does the crash handler pop.
[output_log.txt](https://github.com/tsunamayo/Skywanderers/files/3243749/output_log.txt)

And no I've not had this issue with any other game. I'll try to see if I have any other Unity based games in my Steam library and test to see if they have any issue.

**Comment by Tommyg930** [comment]
Ok I've tested a few other Unity games; Osiris: new Dawn, Kerbal space program, Subnautica and Shadowrun returns. none of them have any issues.

**Comment by grumpysqirl** [comment]
New Backer,  same problem with rtx 2080ti.  CTD when loading and or hanging on loading.

I have no other issues on other Unity games either.

**Comment by tsunamayo** [comment]
@grumpysqirl @Tommyg930 
So you can check 19w23d, bug should be fixed! Tell me if it works.
Thanks for your patience! Cheers

**Comment by grumpysqirl** [comment]
I was able to load in now.
